misk/src/main/kotlin/misk/web/Response.kt

```diff
@@ -48,7 +48,7 @@ fun Response<ResponseBody>.writeToJettyResponse(jettyResponse: HttpServletRespon
 
 private fun HttpServletResponse.bufferedSink() = Okio.buffer(Okio.sink(outputStream))
 
-fun okhttp3.Response.toMisk() : Response<*> {
+fun okhttp3.Response.toMisk() : Response<ResponseBody> {
```

Comment:
I don't think this is correct - further upstream we might not have converted to a `ResponseBody` yet

Reply:
Added better type safety so that if okhttp3.Response has null ResponseBody and empty body is returned instead of null.